resources/applicationautoscaling_policies.go

```diff
@@ -0,0 +1,153 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling"
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func ApplicationautoscalingPolicies() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_applicationautoscaling_policies",
+		Description:  "Information about a scaling policy to use with Application Auto Scaling",
+		Resolver:     fetchApplicationautoscalingPolicies,
+		Multiplex:    client.AccountRegionNamespaceMultiplex,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
```

Comment:
are we sure ARN is enough here? guessing same policy can be on multiple resources in different namespaces?

Reply:
ARN contains the policy name as well. Real data:

```
arn                                          | arn:aws:autoscaling:us-east-1:<ACCOUNT>:scalingPolicy:<UUID>:resource/dynamodb/table/<TABLE>:policyName/DynamoDBReadCapacityUtilization:table/<TABLE>
name                                         | DynamoDBReadCapacityUtilization:table/<TABLE>
type                                         | TargetTrackingScaling
resource_id                                  | table/<TABLE>
scalable_dimension                           | dynamodb:table:ReadCapacityUnits
```

Reply:
Never trust AWS that is my motto